CHANGELOG.md

```diff
@@ -71,59 +66,53 @@ Refactoring SmartC:
 - Removed `getErrorString()` from posixCall, please use `getHumanReadableErrnum()` instead.
 - Enhanced posixCall to handle a common case were multiple errnos are ignored just to suppress error logging
 
-```cpp
-// before
-#include "iceoryx_utils/cxx/smart_c.hpp"
-
-auto unlinkCallPublisher = iox::cxx::makeSmartC(
-    unlink, iox::cxx::ReturnMode::PRE_DEFINED_ERROR_CODE, {ERROR_CODE}, {ENOENT}, sockAddrPublisher.sun_path);
-
-    if (unlinkCallPublisher.hasErrors())
-    {
-        std::cout << "unlink error" << std::endl;
-        exit(1);
-    }
-
-// after
-#include "iceoryx_utils/posix_wrapper/posix_call.hpp"
-
-iox::posix::posixCall(unlink)(sockAddrPublisher.sun_path)
-    .failureReturnValue(ERROR_CODE)
-    .ignoreErrnos(ENOENT, EBUSY) // can be a comma-separated list of errnos
-    .evaluate()
-    .or_else([](auto& r) {
-        std::cout << "unlink error " << r.getHumanReadableErrnum() << std::endl;
-        exit(1);
-    });
+```diff
```

Comment:
I tried here a different approach to make the API changes visible, click in the right menu on `View file` to see the rendered output of this.

Reply:
@elBoberido I disagree, for me this make it visually clearer. For the color-blind folks we could re-introduce the `//after` and `//before`. When I merge something with VS Code it looks [similar](https://code.visualstudio.com/assets/docs/editor/versioncontrol/merge-conflict.png).

Reply:
@mossmaurice well, I kind of can't see the similarities. No red/green. No +/-. But instead code highlighting and something similar to before/after ;)
![grafik](https://user-images.githubusercontent.com/56729607/132373054-2e846ce4-0c78-4cf0-9b05-8f44e95d0484.png)

Reply:
I agree that this isn't the best solution for color-blind people therefore I reverted the changes.